custom/aaa/views.py

```diff
@@ -152,3 +167,38 @@ def post(self, request, *args, **kwargs):
                 parent_id=loc.parent.location_id if loc.parent else None
             ) for loc in locations]
         })
+
+
+@method_decorator([login_and_domain_required, require_superuser_or_contractor], name='dispatch')
+class AggregationScriptPage(BaseDomainView):
+    page_title = 'Aggregation Script'
+    urlname = 'aggregation_script_page'
+    template_name = 'icds_reports/aggregation_script.html'
+
+    @use_daterangepicker
+    def dispatch(self, *args, **kwargs):
+        if settings.SERVER_ENVIRONMENT == 'softlayer':
```

Comment:
should this be `!=`?

Reply:
thanks. fixed https://github.com/dimagi/commcare-hq/pull/23345/commits/d9c827c48dfbdef6d1ba6886d98ed7cf0f2d2049